Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml

```diff
@@ -0,0 +1,35 @@
+- type: entity
+  parent: SeedBase
+  name: pack of dionea spores
+  id: NymphSeed
+  components:
+    - type: Seed
+      seedId: NymphSeed
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/nymph_seed.rsi
+
+
+- type: seed
```

Comment:
this doesn't need to be it's own file. Put the seed entity with the other ones and the seed prototype with the rest of the seed prototypes.